analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -49,6 +49,30 @@ void Test_2(bool condition)
             M2(o.ToString()); // Noncompliant
         }
 
+        void Test_Property()
+        {
+            MyClass o = null;
+            _ = o.MyProperty;   // Noncompliant
+            o = null;
+            o.MyProperty = "";  // Noncompliant
+        }
+
+        void Test_Field()
+        {
+            MyClass o = null;
+            _ = o.MyField;  // Noncompliant
+            o = null;
+            o.MyField = ""; // Noncompliant
+        }
+
+        void Test_Event()
+        {
+            MyClass o = null;
+            o.MyEvent += null;  // Noncompliant
```

Comment:
Does this raise because `o` is null, or because `null` is `null`?
As I assume the 1st one is correct, add some handler instead of the `null` please